test/BaseFixture.sol

```diff
@@ -114,6 +124,29 @@ contract BaseFixture is Test {
         safe.enableModule(address(delayModule));
         safe.enableModule(address(rolesModule));
 
+        // registering the task in CL automation service
+        deal(LINK, address(safe), LINK_FOR_TASK_TOP_UP);
+        IERC20(LINK).approve(address(CL_REGISTRAR), LINK_FOR_TASK_TOP_UP);
+
+        IKeeperRegistrar.RegistrationParams memory registrationParams = IKeeperRegistrar.RegistrationParams({
+            name: string.concat(roboModule.name(), _addressToString(address(safe))),
```

Comment:
still needs a `-` in between no?

Reply:
true, added in e2e465084279af5cb7750f06ca5e9631ddc15ed0